news/2023/2023-05-04-osu-originals-osu-taiko-world-cup-2023.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: "osu! originals - osu!taiko World Cup 2023"
+date: 2023-05-04 11:00:00 +0000
+---
+
+Japan won the [osu!taiko World Cup](/wiki/Tournaments/TWC) for the 10th year in a row, but this is the first TWC with **5 brand-new songs** created specifically for the tournament!
+
+![](/wiki/shared/news/banners/featured-artist.jpg)
+
+We call those songs [**osu! originals**](/wiki/osu!_originals) because... they're original songs made for osu!. While our names lack creativity, the creativity in new songs from [2023's osu!taiko World Cup](/wiki/Tournaments/TWC/2023) make up for it.
```

Comment:
```suggestion
We call those songs [**osu! originals**](/wiki/osu!_originals) because... they're original songs made for osu!. While our names lack creativity, the creativity in the new songs from [2023's osu!taiko World Cup](/wiki/Tournaments/TWC/2023) make up for it.
```